src/ZendCodingStandard/Sniffs/Arrays/FormatSniff.php

```diff
@@ -0,0 +1,233 @@
+                        if ($tokens[$next - 1]['code'] === T_WHITESPACE) {
+                            $phpcsFile->fixer->replaceToken($next - 1, '');
+                        }
+                        $phpcsFile->fixer->addNewlineBefore($next);
+                        $phpcsFile->fixer->endChangeset();
+                    }
+                }
+            } else {
+                if ($previousLine < $tokens[$next]['line'] - 1
+                    && (! empty($tokens[$stackPtr]['conditions'])
+                        || $previousLine === $tokens[$bracketOpener]['line'])
+                ) {
+                    $firstOnLine = $phpcsFile->findFirstOnLine([], $next, true);
+
+                    $error = 'Blank line is not allowed here.';
+                    $fix = $phpcsFile->addFixableError($error, $firstOnLine - 1, 'BlankLine');
+
+                    if ($fix) {
+                        $phpcsFile->fixer->replaceToken($firstOnLine - 1, '');
+                    }
+                }
+
+                $expected = $indent + $this->indent;
+
+                // Check array element indent.
+                if ($firstOnLine = $phpcsFile->findFirstOnLine(T_WHITESPACE, $next)) {
+                    $found = strlen($tokens[$firstOnLine]['content']);
+                    if ($found !== $expected) {
+                        $error = 'Invalid array element indent - expected %d spaces; %d found';
+                        $data = [
+                            $expected,
+                            $found,
+                        ];
+                        $fix = $phpcsFile->addFixableError($error, $firstOnLine, 'ElementIndent', $data);
+
+                        if ($fix) {
+                            $phpcsFile->fixer->replaceToken($firstOnLine, str_repeat(' ', $expected));
+                        }
+                    }
+                } else {
+                    $error = 'Invalid array element indent - expected %d spaces; 0 found';
+                    $data = [$expected];
+                    $fix = $phpcsFile->addFixableError($error, $next, 'ElementIndent', $data);
+
+                    if ($fix) {
+                        $phpcsFile->fixer->addContentBefore($next, str_repeat(' ', $expected));
+                    }
+                }
+            }
+
+            if ($tokens[$next]['code'] === T_COMMENT
+                && (strpos($tokens[$next]['content'], '//') === 0
+                    || strpos($tokens[$next]['content'], '#') === 0)
+            ) {
+                $end = $next;
+            } else {
+                $end = $phpcsFile->findEndOfStatement($next);
+                if ($tokens[$end]['code'] === T_DOUBLE_ARROW) {
+                    $end = $phpcsFile->findEndOfStatement($end);
+                }
+            }
+
+            $previousLine = $tokens[$end]['line'];
+            $next = $end;
+        }
+
+        if ($first = $phpcsFile->findFirstOnLine([], $bracketCloser, true)) {
```

Comment:
I think it is refactored, and the comment was added on previous version. It's outdated right now.